packages/core/src/percy.js

```diff
@@ -404,6 +415,22 @@ export class Percy {
     if (syncMode) options.sync = syncMode;
     return syncMode;
   }
+
+  sendBuildEvents() {
+    // Only send cli error when PERCY_CLIENT_ERROR_LOGS is set to true
+    if (process.env.PERCY_CLIENT_ERROR_LOGS !== 'false') {
```

Comment:
`PERCY_CLIENT_ERROR_LOGS` This is for the CI errors of users?

Reply:
Yes